### DISCUSSION TITLE
Server Stats/Gui

### DISCUSSION THREAD

**Discussion by vacav88** [mbround18/valheim-docker#203] [Ideas]
Loving the container and your work so far, really thankful for it also!

Not sure if its been requested already but I'd love some way to see some basic statistics about the server. 
The main one being the number of players currently online. (Unless I'm missing a way to do this easily)

I'd also love it if we could have that info as well as some basic commands available on a website gui. Such as start/stop and the upcoming update command. It's quite tricky for me to do these remotely on a phone so having a basic page where I could simply click a button to fix an issue with the server would be amazing.

**Comment by mbround18** [comment]
This is a work in progress for player status, see https://github.com/mbround18/valheim-docker/issues/135

As for a web gui, take a look at https://github.com/SelfhostedPro/Yacht

**Reply by vacav88** [reply]
Can't believe I missed #135 I'll keep an eye on that.

Awesome ill have a look into that as well

**Comment by Aldjinn** [comment]
Have a look at BattleMetrics [1] as an interim solution until #135 is finished.

[1] https://www.battlemetrics.com/servers/valheim

**Reply by mbround18** [reply]
Yee they just hit your query port from a public game listing :) love battle metrics.

**Reply by vacav88** [reply]
Thanks @Aldjinn - That's a perfect solution for the time being!

**Reply by Aldjinn** [reply]
Otherwise, I've you would like to run something like that on your own, give [node-gamedig](https://github.com/gamedig/node-gamedig) a try.

Example:
```bash
npm install gamedig -g
gamedig --pretty --type valheim my.valheim-server.com:2456
```
